eng/references.targets

```diff
@@ -1,33 +1,43 @@
 <Project>
   <PropertyGroup>
-    <ContractDependencyPaths>$(RefPath)</ContractDependencyPaths>
+    <IncludeDefaultReferences Condition="'$(IncludeDefaultReferences)' == ''">true</IncludeDefaultReferences>
+    <IncludeNETStandardReference Condition="'$(IncludeDefaultReferences)' == 'true' and
```

Comment:
Why do we need to include netstandard when TFI != .NETStandard?

Reply:
That was added by @weshaggard here: https://github.com/dotnet/buildtools/pull/1460/files#diff-8f8878b439291dd5d102ff6d2414ce67R59-R62. AFAIK this is to combine different type universes (System.Runtime vs netstandard vs mscorlib). cc @ericstj

Reply:
Yeah, if we want to allow implementations that target netcoreapp to be able to reference netstandard assemblies then we need the netstandard facades.